app/models/organization_webhook.rb

```diff
@@ -90,6 +90,7 @@ def activate_org_hook(client)
   # of a large size. Invoke cautiously.
   def ensure_webhook_is_active!(client: nil)
     client ||= admin_org_hook_scoped_github_client
+    retrieve_org_hook_id!(client) if github_id.blank?
```

Comment:
To reiterate: if the `github_id` field on the record is blank, we double check that the webhook doesn't;t already exist.